heroku/cli
# feat(cli): add cli performance telemetry

#### This PR completes the following
[Instrument Heroku CLI Node process with OpenTelemetry](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001U07StYAJ/view)
[Integrate CLI with Backboard OTel collector](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001WzrWqYAJ/view)

#### NOTE
This work will roll up into the [zw/cli-telemetry](https://github.com/heroku/cli/tree/zw/cli-telemetry) feature branch.

#### How to Verify
- Pull down branch `zw/telemetry-integration`
- Navigate to `/packages/cli`
- Successfully run command locally
   - (I've been using `./bin/run pipelines:info pipeline-practice`)
- Confirm telemetry trace in our `front-end-metrics-development` branch in honeycomb
- Abort command while running locally (Use `CTRL+C` while command is still running)
   -  `./bin/run pipelines:info pipeline-practice`
- Confirm telemetry `Received SIGINT` trace in our `front-end-metrics-development` branch in honeycomb
